### ISSUE TITLE
Fix prop-types errors

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#14837]
After merging https://github.com/forem/forem/pull/14635, we have a lot of prop-types errors in the console that need to be cleaned up. These errors can be observed throughout the onboarding flow and during general app usage when running the app locally.

Each error will need to be investigated to check whether the `component.propTypes` declaration needs to be changed, or whether the prop being passed to the given component needs to be changed.

We can/should complete this work in multiple small PRs (e.g. per page).

**Comment by papuruth** [comment]
Hi I would like to work on this.

**Comment by cmgorton** [comment]
Great @papuruth . I assigned it to you. Let us know if you have any questions.

**Comment by papuruth** [comment]
@cmgorton I have started the application using bin/start but could you tell how to login. 
Also the application is running on port 3000 is this correct?

**Comment by cmgorton** [comment]
Hey @papuruth . I see you have a PR started. Where you able to login and get the application running? Or did you still need help with that?

**Comment by papuruth** [comment]
> Hey @papuruth . I see you have a PR started. Where you able to login and get the application running? Or did you still need help with that?

No, no help is required as I was able to log in using Github. Thanks btw.
